src/main/java/com/thebois/controllers/infocontrollers/InfoController.java

```diff
@@ -0,0 +1,61 @@
+package com.thebois.controllers.infocontrollers;
+
+import java.util.ArrayList;
+import java.util.Collection;
+
+import com.badlogic.gdx.scenes.scene2d.ui.Skin;
+
+import com.thebois.controllers.IController;
+import com.thebois.models.world.World;
+import com.thebois.views.infoviews.IActorView;
+import com.thebois.views.infoviews.InfoView;
+
+/**
+ * Container class for controllers that manage the info.
+ */
+public class InfoController {
```

Comment:
That would require me to give and interface to 

> Another thing, you should use the built-in frame exporter of Miro to create images of the package class diagrams. The larger ones can't be read in the images you posted because of too low resolution in the screenshot.

I did that and chose medium size on the export, should I redo it with large instead?

Reply:
It does work, and probably should have done it. Changing it for both Info- and Worldcontroller